core/library.py

```diff
@@ -908,7 +909,7 @@ def searchresults(guid, status, movie_info=None):
 
             search_result = searchresults.score([search_result], imported=True)[0]
 
-            required_keys = ('score', 'size', 'status', 'pubdate', 'title', 'imdbid', 'indexer', 'date_found', 'info_link', 'guid', 'torrentfile', 'resoluion', 'type', 'downloadid', 'freeleech')
+            required_keys = ('score', 'size', 'status', 'pubdate', 'title', 'imdbid', 'indexer', 'date_found', 'info_link', 'guid', 'torrentfile', 'resolution', 'type', 'downloadid', 'freeleech')
```

Comment:
Don't go looking for typos, you'll quickly be overwhelmed. Too bad IDEs don't have good spell checkers.

Reply:
Sure typo may not be the priority but in this case it has an impact on what is stored in the DB and thus loose info that is never recovered later on

Reply:
I'm not saying it doesn't need to be fixed, only that finding all of them would take a full day.

And its *lose*.